### ISSUE TITLE
Is this still being worked on?

### ISSUE THREAD

**Issue by six-shooter** [tanepiper/elite-dangerous-hue-scripter#2]
I would love to know if this is still being worked on, as I would love to integrate my Hue lights into a cockpit, but have no idea about JS or linux commands

**Comment by Deadweasel** [comment]
Yeah kinda think it was a set-it-and-forget-it project.
I'm not very familiar with the setup process used here either, but the instructions should have been good enough to get things working for me.  Sadly, I get a bunch of "deprecated" mentions and other failures, meaning some of the things this project relies on have themselves been updated to the point where this likely wouldn't even work, even if I could get the initial setup stuff to complete.  :(

**Comment by tanepiper** [comment]
@six-shooter @Deadweasel Yes this project isn't active - I mostly stopped playing ED and didn't continue to work on it - it's mostly here for the reference of the underlying code for reading ED data and sending to Hue lights.
